app/agent/origin.scala

```diff
@@ -56,20 +61,57 @@ trait Origin extends Marker {
 }
 
 case class Credentials(accessKey: Option[String], role: Option[String], profile: Option[String], region: String)(secretKey: Option[String]) {
```

Comment:
This case class has ballooned because we're currently running both v1 and v2 of the SDK at the same time. This will look much better once we move all the collectors onto v2!😅